test/common/http/http2/conn_pool_test.cc

```diff
@@ -1436,30 +1436,46 @@ TEST_F(Http2ConnPoolImplTest, PreconnectWithoutMultiplexing) {
 
 TEST_F(Http2ConnPoolImplTest, DisconnectWithNegativeCapacity) {
   TestScopedRuntime scoped_runtime;
-  cluster_->http2_options_.mutable_max_concurrent_streams()->set_value(2);
+  cluster_->http2_options_.mutable_max_concurrent_streams()->set_value(4);
   ON_CALL(*cluster_, perUpstreamPreconnectRatio).WillByDefault(Return(1));
 
   // One stream results in one connection. Two streams result in two connections.
```

Comment:
Also not yours, but I think this comment is from another test and should be removed here (as long as I'm looking at it)b